plugin/keyring_vault/CMakeLists.txt

```diff
@@ -7,23 +18,13 @@ SET(CMAKE_EXTRA_INCLUDE_FILES)
 MACRO (CHECK_IF_LIB_FOUND lib_name project_name)
   SET (lib_found_variable_name "${lib_name}_FOUND")
   IF (NOT DEFINED ${lib_found_variable_name} OR NOT ${${lib_found_variable_name}})
-    message("Not building ${project_name}, could not find library: ${lib_name}")
-    RETURN()
-  ENDIF()
-ENDMACRO()
-
-MACRO (CHECK_IF_CURL_DEPENDS_ON_RTMP project_name)
-  EXECUTE_PROCESS(COMMAND ldd ${CURL_LIBRARY}
-                  COMMAND grep rtmp
-                  OUTPUT_VARIABLE CURL_DEPENDS_ON_RTMP)
-  IF (NOT CURL_DEPENDS_ON_RTMP STREQUAL "")
-    message("Not building ${project_name}. The supplied CURL library depends on rtmp library.
-Please provide CURL library that does not depend on rtmp library to build keyring_vault unittests.")
+    message(${KEYRING_VAULT_STATUS_MODE} "Not building ${project_name}, could not find library: ${lib_name}")
```

Comment:
This macro has a generic name (`CHECK_IF_LIB_FOUND`), takes a `project_name` argument (called only with `"keyring_vault"`), but the message uses the fixed `KEYRING_VAULT_STATUS_MODE` variable. 

Isn't this a bit inconsistent/confusing? Especially since macros are available in the parents cmakelists too after their definition.

Reply:
Done, made status a parameter.